File: HackerRank Solutions/Problem Solving/Mathematics/Algebra/Number Groups/Number Groups.java
import java.io.*;
import java.math.*;
import java.security.*;
import java.text.*;
import java.util.*;
import java.util.concurrent.*;
import java.util.regex.*;
public class Solution {

    // Complete the sumOfGroup function below.
    static long sumOfGroup(int k) {
        // Return the sum of the elements of the k'th group.
        return (long)k*k*k;
    }

    private static final Scanner scanner = new Scanner(System.in);

    public static void main(String[] args) throws IOException {
        BufferedWriter bufferedWriter = new BufferedWriter(new FileWriter(System.getenv("OUTPUT_PATH")));

        int k = scanner.nextInt();
        scanner.skip("(\r\n|[\n\r\u2028\u2029\u0085])?");

        long answer = sumOfGroup(k);

        bufferedWriter.write(String.valueOf(answer));
        bufferedWriter.newLine();

        bufferedWriter.close();

        scanner.close();
    }
}

File: HackerRank Solutions/Problem Solving/Mathematics/Algebra/Number Groups/Number Groups.cpp
#include <bits/stdc++.h>

using namespace std;

// Complete the sumOfGroup function below.
long sumOfGroup(int k) {
    // Return the sum of the elements of the k'th group.
    return (long)k*k*k;
}

int main()
{
    ofstream fout(getenv("OUTPUT_PATH"));

    int k;
    cin >> k;
    cin.ignore(numeric_limits<streamsize>::max(), '\n');

    long answer = sumOfGroup(k);

    fout << answer << "\n";

    fout.close();

    return 0;
}
